Remotion/ObjectBinding/Web.Development.WebTesting/ControlObjects/BocAutoCompleteReferenceValueControlObject.cs

```diff
@@ -392,6 +391,24 @@ private IReadOnlyList<SearchServiceResultItem> ParseSearchServiceResponse ([NotN
       }
     }
 
+    private static IReadOnlyList<SearchServiceResultItem> ParseSearchServiceResultVariants (IDictionary<string, object> data) =>
```

Comment:
Since there's an update anyway: IReadOnlyDictionary would be great instead of the IDictionary casts.